tests/test_embedding.py

```diff
@@ -0,0 +1,63 @@
+import finalfusion
+import pytest
+import numpy
+
+TEST_NORMS = [
+    6.557438373565674,
+    8.83176040649414,
+    6.164413928985596,
+    9.165151596069336,
+    7.4833149909973145,
+    7.211102485656738,
+    7.4833149909973145
+]
+
+
+def test_embeddings_with_norms():
+    embeds = finalfusion.Embeddings(
+        "tests/embeddings.fifu")
+    embeds_dict = dict()
+    with open("tests/embeddings.txt", "r", encoding="utf8")as lines:
+        for line in lines:
+            line_list = line.split(' ')
+            embeds_dict[line_list[0]] = [float(val) for val in line_list[1:]]
+
+    for each_pair in zip(embeds.iter_with_norm(), TEST_NORMS):
```

Comment:
Sorry for nitpicking more, but:

```
for embedding_with_norm, norm in zip(embeds.iter_with_norm(), TEST_NORMS):
```

Is clearer, since it gives one level less to index on.

Reply:
Thank you very, very much for constantly correcting me! I'm sorry for bothering you like an idiot these days, but I have learned a lot and I'm still learning. I'm sure I'll get better! :)

Reply:
No problem. We all learn all the time :).